Havit.Blazor.Components.Web.Bootstrap.Documentation/Pages/Components/HxCalendarDoc/HxCalendar_Documentation.razor

```diff
@@ -20,6 +20,27 @@
             <code>HxCalendar</code> is used in implementation of <a href="/components/HxInputDate"><code>HxInputDate</code></a>
             and <a href="/components/HxInputDateRange"><code>HxInputDateRange</code></a> components. See their documentation for more info.
         </p>
+
+        <DocHeading Title="TimeZones" />
+        <p>
+            You can change the timezone used by the calendar by providing a custom <a href="https://devblogs.microsoft.com/dotnet/announcing-dotnet-8-preview-4/#introducing-time-abstraction"><code>TimeProvider</code></a>.
+            TimeProvider is resolved in the following order:
+            <ol>
+                <li>TimeProvider parameter on <code>HxCalendar</code>, <a href="/components/HxInputDate"><code>HxInputDate</code></a> and <a href="/components/HxInputDateRange"><code>HxInputDateRange</code></a>.</li>
+                <li>Settings <see cref="HxCalendar.Defaults" /></li>
+                <li>Defaults <see cref="HxCalendar.Defaults" /></li>
```

Comment:
1. Use `code` HTML elements for all class/member names: `TimeProvider`, `Settings`, `Defaults`, ...
2. Do not use `<see cref` here as we do not have any processing for such markup, use `<a href` instead